pkg/model/crd.go

```diff
@@ -626,6 +626,94 @@ func (r *CRD) addAdditionalPrinterColumns(additionalColumns []*ackgenconfig.Addi
 	}
 }
 
+func (crd *CRD) checkSpecOrStatus(
+	field string,
+) (*Field, bool) {
+	fSpec, okSpec := crd.SpecFields[field]
+	if okSpec {
+		return fSpec, okSpec
+	}
+	fStatus, okStatus := crd.StatusFields[field]
+	if okStatus {
+		return fStatus, okStatus
+	}
+	return nil, false
+}
+
+// addCustomNestedFields injects the provided customNestedFields into the
+// Spec or Status struct as a nested field. The customNestedFields are
+// identified by the field path. The field path is a dot separated string
+// that represents the path to the nested field. For example, if we want to
+// inject a field called "Password" into the "User" struct, the field path
+// would be "User.Password". The field path can be as deep as needed.
+func (crd *CRD) addCustomNestedFields(customNestedFields map[string]*ackgenconfig.FieldConfig) {
+	// Now that we collected all the nested fields, we can process, validate
```

Comment:
What does "now" mean here? The message doesn't seem relevant in the beginning of a function.